deploy/devdeploy.py

```diff
@@ -64,37 +63,31 @@ def devdeploy(profile: str, database: str, schema: str, stage: str):
 
 
 def usage():
-    print(
-        "devdeploy.py -p <snowsql_profile_name> [(-d | --database=) <database_name>] [(-s | --stage=) <stage_name>]"
-    )
+    print("devdeploy.py -p <snowsql_profile_name> [(-s | --stage=) <stage_name>]")
```

Comment:
I don't think we need to specify the stage name. Its as much set as the schemas. The only reason its an option in helpers is because in dev deploy the stage has to have a name whereas in prod its taken from the default stage (and the stage arg should be an empty string from prod deploy)

Reply:
done